### PR TITLE
FOGL-3212: set warning message instead of error for missing plugin directories

### PR DESCRIPTION
**Pull request by pintomax**
FOGL-3212: set warning message instead of error for missing plugin directories

### CODE REVIEW

**Review by ashish-jabble — CHANGES_REQUESTED** [review]
@pintomax As per Mark comments on JIRA
```
Mark Riddoch
August 27, 2019, 2:31 PM

We should really create the empty plugin directories in the install phase of the notification service.
```

**Comment by pintomax** [comment]
@ashish-jabble I've also added creation of create the empty plugin directories in 

https://github.com/fledge-iot/fledge-service-notification/pull/18

File: C/services/common/plugin_manager.cpp [hunk 1]
```diff
@@ -569,7 +569,7 @@ void PluginManager::getInstalledPlugins(const string& type,
 			// Can not open specified dir path
 			char msg[128];
 			char* ret = strerror_r(errno, msg, 128);
-			logger->error("Can not access plugin directory %s: %s",
+			logger->warn("Can not access plugin directory %s: %s",
```
Comment (ashish-jabble) [inline C/services/common/plugin_manager.cpp:572]: @pintomax You have made a comment on JIRA
```
changed log level to debug (Fledge side)
```

Comment (pintomax) [reply C/services/common/plugin_manager.cpp:572]: @ashish-jabble Fixed comment: it's now warning in the JIRA comment

**Review by ashish-jabble — APPROVED** [review]

**Pull request merged**